Resources/Prototypes/Entities/Clothing/Ears/headsets.yml

```diff
@@ -13,32 +20,59 @@
     - ears
     sprite: Clothing/Ears/Headsets/base.rsi
 
+- type: entity
+  parent: ClothingHeadset
+  id: ClothingHeadsetGrey
+  name: passenger headset
+  description: An updated, modular intercom that fits over the head. Takes encryption keys.
+  components:
+  - type: ContainerFill
+    containers:
+      key_slots:
+      - EncryptionKeyCommon
+  - type: Sprite
+    sprite: Clothing/Ears/Headsets/base.rsi
+
 - type: entity
   parent: ClothingHeadset
   id: ClothingHeadsetCargo
   name: cargo headset
   description: A headset used by supply employees.
   components:
-  - type: Headset
-    channels:
-    - Common
-    - Supply
+  - type: ContainerFill
+    containers:
+      key_slots:
+      - EncryptionKeyCargo
   - type: Sprite
     sprite: Clothing/Ears/Headsets/cargo.rsi
   - type: Clothing
     sprite: Clothing/Ears/Headsets/cargo.rsi
 
+- type: entity
+  parent: ClothingHeadset
+  id: ClothingHeadsetMining
+  name: mining headset
+  description: Headset used by shaft miners.
+  components:
+  - type: ContainerFill
+    containers:
+      key_slots:
+      - EncryptionKeyCargo
+  - type: Sprite
+    sprite: Clothing/Ears/Headsets/mining.rsi
+  - type: Clothing
+    sprite: Clothing/Ears/Headsets/mining.rsi
+
```

Comment:
why did you add a headset for shaft miners if the job doesn't exist?

Reply:
![image](https://user-images.githubusercontent.com/46600554/211192190-0c79fbd1-8a75-4c29-b9d3-809fab9723ba.png)
I didn't add it, i move it to cargo headsets